Assets/csc.rsp

```diff
@@ -0,0 +1 @@
+-warnaserror+
```

Comment:
Is it possible to pass this via our msbuild scripts instead of adding this file?

Reply:
It is, though this means that people will catch this at PR CI time, rather than in their inner loop.

Reply:
https://www.reddit.com/r/Unity3D/comments/adfxna/unity_developer/